src/main/java/org/terasology/las/LaSSpawner.java

```diff
@@ -15,7 +15,7 @@
 
     @Override
     public Vector3f getSpawnPosition(World world, EntityRef clientEntity) {
-        return new Vector3f(new Vector3i(JomlUtil.from(LASUtils.FLOATING_PLATFORM_POSITION))
+        return new Vector3f(new Vector3i(LASUtils.FLOATING_PLATFORM_POSITION)
                 .add(0, 5, 0));
```

Comment:
Can we get rid of the intermediate `Vector3i` here?

```suggestion
        return new Vector3f(LASUtils.FLOATING_PLATFORM_POSITION).add(0, 5, 0);
```